pymeasure/instruments/kepco/kepcobop.py

```diff
@@ -0,0 +1,180 @@
+#
+# This file is part of the PyMeasure package.
+#
+# Copyright (c) 2013-2024 PyMeasure Developers
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+
+# List of Kepco BOP's (Vmax,Imax) [single channel output]
+# 100 W:
+# 5-20
+# 20-5
+# 50-2
+# 100-1
+
+# 200 W:
+# 5-30
+# 20-10
+# 36-6
+# 50-4
+# 72-3
+# 100-2
+# 200-1
+
+# 400 W:
+# 20-20
+# 36-12
+# 50-8
+# 72-6
+# 100-4
+
+from pymeasure.instruments import Instrument, SCPIMixin
+from pymeasure.instruments.validators import strict_discrete_set, \
+    truncated_range
+
+OPERATING_MODES = ['VOLT', 'CURR']
+
+
+class KepcoBOP3612(SCPIMixin, Instrument):
+    """
+    Represents the Kepco BOP 36-12 (M or D) 400 W bipolar power supply
+    fitted with BIT 4886 digital interface card (minimal implementation)
+    and provides a high-level interface for interacting with the instrument.
+    """
+    _Vmax = 36
+    _Imax = 12
+
+    def __init__(self, adapter, name="Kepco BOP 36-12 Bipolar Power Supply",
+                 baud_rate=9600, read_termination="\n", write_termination="\n",
```

Comment:
Can you change the baud_rate and the termination of the instrument?
If not, you should hard code them in the super().__init__() call below.

Reply:
No, it looks like it can't be changed. Thanks.